plugins/renepay/pay.h

```diff
@@ -0,0 +1,121 @@
+#ifndef LIGHTNING_PLUGINS_RENEPAY_PAY_H
+#define LIGHTNING_PLUGINS_RENEPAY_PAY_H
+#include "config.h"
+#include <ccan/list/list.h>
+#include <common/node_id.h>
+#include <plugins/libplugin.h>
+#include <plugins/renepay/flow.h>
+#include <plugins/renepay/payment.h>
+
+// TODO(eduardo): renepaystatus should be similar to paystatus
```

Comment:
... Actually, I'd like it to be more informative!  Don't feel bound!

Reply:
With time we will add more information. Right now I saw we needed to show the estimated prob. of success of individual flows: c754f88a79993ae1ab2f20a4ce951c844eefe589

Reply:
I'll also add the total probability of set of flows, which sometimes is not necessarily the multiplication of individual flows, f2a65458b2eefd314c78302f640824244dcfa89a.
BTW if there are pending HTLCs these probabilities have to be interpreted as conditional probabilities: if all the pending HTLCs succeed then this is the prob. of success of this batch of flows..